packages/mantine/babel.config.js

```diff
@@ -0,0 +1,3 @@
+const defaultConfig = require('../../babel.config');
```

Comment:
You will need to convert this to a `.json`  file due to the `package.json` changes I'm requesting